packages/connectivity/src/scp-cf/destination/destination-cache.spec.ts

```diff
@@ -296,15 +287,21 @@ describe('destination cache', () => {
 
     const destName = destinationOne.name!;
 
-    it('disables the cache by default', async () => {
+    it('uses the cache by default', async () => {
```

Comment:
[pp] I would place this test at the top.

Reply:
Done.